### PR TITLE
Fix bug in services framework keys

### PR DESCRIPTION
**Pull request by lfdebrux**
PR #28 updated the framework keys for brief object stubs, but also accidentally changed the framework keys for service object stubs. We didn't notice at the time because we didn't update dmtestutils in the api repo, but this change breaks the tests for the api app.

This commit changes the framework keys in service object stubs to be back inline with the current model, and also tweaks the briefs frameworks keys code to be more explicit.

### CODE REVIEW

File: dmtestutils/api_model_stubs/brief.py [hunk 1]
```diff
@@ -65,10 +65,14 @@ def __init__(self, **kwargs):
             and ("framework_name" not in kwargs and "frameworkName" not in kwargs)
             and ("framework_family" not in kwargs and "frameworkFramework" not in kwargs)
         ):
-            self.response_data.update(self._format_framework(self.response_data["frameworkSlug"], oldstyle=True))
+            # The brief model serialisation includes both new-style and old-style framework keys,
+            # and the deprecated `frameworkFramework` field. Consumers should use `brief['framework']['family']`,
+            # not `brief['frameworkFamily']` or `brief['frameworkFramework]`, but we should still make sure
+            # our test stubs reflect the model.
+            self.response_data.update(
+                self._format_framework(self.response_data["frameworkSlug"], newstyle=True, oldstyle=True)
```
Comment (gidsg) [inline dmtestutils/api_model_stubs/brief.py:73]: tiny nitpick but would `old_style` and `new_style` be more idiomatic python?

Comment (lfdebrux) [reply dmtestutils/api_model_stubs/brief.py:73]: I'm happy to change it, although I should note that underscores aren't necessarily required, PEP8 just says variable names should be lowercase

> with words separated by underscores as necessary to improve readability

https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Do you find it harder to read the names as is?

Comment (gidsg) [reply dmtestutils/api_model_stubs/brief.py:73]: It is not a big deal, intellij will flag it as a typo as-is though.

**Review by gidsg — APPROVED** [review]
Good spot 🕵️ 👍

**Pull request merged**